contrib/crio-wipe/crio-wipe.bash

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -eu
+
+dir=${0%/*}
+source "$dir/lib.bash"
+source "$dir/config.bash"
+
+print_usage() {
+	echo "$(basename $0) [-f version-file-location] [-d containers-storage-dir] [-w wipe]"
+}
+
+# process command variables
+while getopts 'f:d:w:h' OPTION; do
```

Comment:
FYI- there are two `getopts`, there's one built-in to bash (the one you're using here) and another `/usr/bin/getopt` (no "s") .  The external one supports full GNU-style command-line conventions, including long options.  However it adds some complexity to the parsing loop.  For what this script is being used for, the simple, built-in is probably fine.